.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,45 @@
+### Description of change
+
+_Please provide description of change here_.
+
+### Pull Request check-list
+
+_Please make sure to review and check all of these items_:
+
+- [ ] Does `make test` pass after this change?
+- [ ] Is commit message formatted according to [CONTRIBUTING.md][0]
+
+### Affected core subsystem(s)
+
+- [ ] buffer
+- [ ] child_process
+- [ ] cluster
+- [ ] crypto
+- [ ] dgram
+- [ ] dns
```

Comment:
Right now we don't have caine, and it should be beneficial to experiment with templates to see what works and what does not.